ocs_ci/helpers/cnv_helpers.py

```diff
@@ -174,30 +174,38 @@ def create_dv(
     pvc_size="30Gi",
     source_url=constants.CNV_CENTOS_SOURCE,
     namespace=constants.CNV_NAMESPACE,
+    source_pvc_name=None,
+    source_pvc_ns=None,
 ):
     """
-    Create a PVC using a specified data source
+    Create/Clones a DV using a specified data source
```

Comment:
Please consider creating a separate function for cloning DVs

Reply:
created a new helper function to handle this